seqr/views/utils/test_utils.py

```diff
@@ -671,28 +668,23 @@ def _assert_airflow_calls(self, dag_name, dag_variables, call_count, secondary_d
             'key': dag_name,
             'value': json.dumps(dag_variables),
         })
-        self.assertEqual(responses.calls[offset+1].request.headers['Authorization'], self.auth_header)
 
         # get task id
         self.assertEqual(responses.calls[offset+2].request.url, f'{dag_url}/tasks')
         self.assertEqual(responses.calls[offset+2].request.method, 'GET')
-        self.assertEqual(responses.calls[offset+2].request.headers['Authorization'], self.auth_header)
 
         self.assertEqual(responses.calls[offset+3].request.url, f'{dag_url}/tasks')
         self.assertEqual(responses.calls[offset+3].request.method, 'GET')
-        self.assertEqual(responses.calls[offset+3].request.headers['Authorization'], self.auth_header)
 
         call_cnt = call_count - 1
         if call_count > 5:
             self.assertEqual(responses.calls[offset+4].request.url, f'{dag_url}/tasks')
             self.assertEqual(responses.calls[offset+4].request.method, 'GET')
-            self.assertEqual(responses.calls[offset+4].request.headers['Authorization'], self.auth_header)
 
         # trigger dag
         self.assertEqual(responses.calls[offset+call_cnt].request.url, f'{dag_url}/dagRuns')
         self.assertEqual(responses.calls[offset+call_cnt].request.method, 'POST')
         self.assertDictEqual(json.loads(responses.calls[offset+call_cnt].request.body), {})
-        self.assertEqual(responses.calls[offset+call_cnt].request.headers['Authorization'], self.auth_header)
```

Comment:
These requests do not have this header anymore, but they are being run from an authenticated session which means *something* is different about them than if they weren't, and we should be testing that if thats at all possible

Reply:
I think it's baked into whatever `AuthorizedSession` is doing under the hood... I do think worth investigating further though.